README.md

```diff
@@ -117,6 +117,12 @@ If the `lang` property is not present in the body of your request, the default e
 
 [Gibbon](https://en.wikipedia.org/wiki/Gibbon) as an allusion to [Mandrill App](https://mandrill.com/).
 
+### I see Gibbon-mail is using [Mozilla's Nunjucks](https://mozilla.github.io/nunjucks/)
```

Comment:
```suggestion
### Customer filters
```
Gibbon-mail is using [Mozilla's Nunjucks](https://mozilla.github.io/nunjucks/), but you can also create your own customer filter.

To do this, refer to [backend/filters/index.js](backend/filters/index.js).